client/src/components/CompleteSetButton.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import ButtonGroup from '@mui/material/ButtonGroup';
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
+import ClickAwayListener from '@mui/material/ClickAwayListener';
+import Grow from '@mui/material/Grow';
+import Paper from '@mui/material/Paper';
+import Popper from '@mui/material/Popper';
+import MenuItem from '@mui/material/MenuItem';
+import MenuList from '@mui/material/MenuList';
+
+enum CompleteSetOptions {
+  COMPLETE = 'COMPLETE',
+  SKIP = 'SKIP',
+  EDIT = 'EDIT',
+}
+
+const completeSetOptionsCopyMap = new Map<CompleteSetOptions, string>([
+  [CompleteSetOptions.COMPLETE, 'Complete set'],
+  [CompleteSetOptions.SKIP, 'Skip set'],
+  [CompleteSetOptions.EDIT, 'Edit set'],
+]);
+
+const options = Object.values(CompleteSetOptions);
+
```

Comment:
With the exception of the use of an enum instead of string for button selection ☝️ , this is 100% copied from the MUI example page: https://mui.com/material-ui/react-button-group/#split-button